packages/dialtone-vue3/directives/tooltip/tooltip.test.js

```diff
@@ -44,7 +44,6 @@ describe('DtTooltipDirective Tests', () => {
         },
         plugins: [DtTooltipDirective],
       },
-      attachTo: document.body,
```

Comment:
Gotta say I am very surprised it doesn't need this since this component renders at the body...

Reply:
You are right, it makes sense to have it in this case. I don't know why the tests pass without it, but I added it back.